src/Core/Grid/Query/WebserviceKeyQueryBuilder.php

```diff
@@ -79,8 +87,10 @@ public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria)
      */
     public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria)
     {
-        $qb = $this->getQueryBuilder($searchCriteria->getFilters());
-        $qb->select('COUNT(wa.`id_webservice_account`)');
+        $qb = $this->getQueryBuilder($searchCriteria->getFilters())
+            ->select('SQL_CALC_FOUND_ROWS')
```

Comment:
Why adding two columns in select result? and what is the gain of using `SQL_CALC_FOUND_ROWS` and `FOUND_ROWS()` instead of `COUNT` here? @jocel1 any idea?

Reply:
I strongly recommend using COUNT when possible

Reply:
i think its better to use 2 queries, we already discussed it in another PR.

Reply:
I don't understand why it's a good idea to have 2 queries instead of one, when having two queries can have performance issues :/

Reply:
Thanks for the insights :+1:  - I managed to make it in just a single sql by separating group by and instead of plain COUNT I used the combination of COUNT and DISTINCT - and indeed,  from reading some benchmarks, FOUND_ROWS is slower then COUNT